install_config/upgrading/blue_green_deployments.adoc

```diff
@@ -0,0 +1,159 @@
+[[upgrading-blue-green-deployments]]
+= Blue-Green Deployments
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Blue-green deployments are a proven approach to reducing downtime caused while
+upgrading an environment. This is done by creating a parallel environment on
+which the new deployment can be installed. If a problem is detected, and after
+the new deployment is verified, traffic can be switched over with the option to
+rollback.
+
+While blue-green is a valid strategy for deploying just about any software,
+there are always trade-offs. Not all environments have the same uptime
+requirements or the resources to properly perform blue-green deployments. In an
+{product-title} environment, the most suitable candidate for blue-green
+deployments are the nodes. All user processes run on these systems and even
+critical pieces of {product-title} infrastructure are self-hosted there. Uptime
+is most important for these workloads and the additional complexity of
+blue-green deployments can be justified. The exact implementation of this
+approach varies based on your requirements. Often the main challenge is having the
+excess capacity to facilitate such an approach.
+
+ifdef::openshift-enterprise[]
+Another lesser challenge is that the administrator must temporarily share the
+Red Hat software entitlements between the blue-green deployments or provide
```

Comment:
Recommended to @ahardin-rh to consider breaking this up, leaving the "lesser challenge" part here and moving the actual steps into "Preparing for Upgrade" below.